File: rhino/src/main/java/com/script/Compilable.java
/*
 * Decompiled with CFR 0.152.
 */
package com.script;

import java.io.Reader;

public interface Compilable {
    public CompiledScript compile(Reader var1) throws ScriptException;

    public CompiledScript compile(String var1) throws ScriptException;
}

File: rhino/src/main/java/com/script/Compilable.kt
/*
 * Decompiled with CFR 0.152.
 */
package com.script;

import java.io.Reader;

@SuppressWarnings("UnnecessaryModifier")
public interface Compilable {
    public CompiledScript compile(Reader var1) throws ScriptException;

    public CompiledScript compile(String var1) throws ScriptException;
}
